Makefile

```diff
@@ -156,6 +156,14 @@ publish-wasm: clean-wasm build-wasm
 test-valgrind: libflux
 	cd libflux/c && $(MAKE) test-valgrind
 
+# Build the set of supported cross-compiled binaries
+flux-release: Dockerfile_build
```

Comment:
Would a better name for this be `test-release`? This doesn't perform a release and it seems like that could make the name confusing.

Reply:
Maybe `test-flux-release`?

Reply:
That's fine for me. I like `test-release` just because you're already in the flux repository so it's not like you're testing another project's release process.

Reply:
Valid point.